addon/components/ember-sparkles/pie-chart.js

```diff
@@ -0,0 +1,25 @@
+import Ember from 'ember';
+import { select } from 'd3-selection';
+import layout from '../../templates/components/ember-sparkles/pie-chart';
+
+export default Ember.Component.extend({
+  layout,
+  tagName: 'g',
+  classNames: ['ember-sparkles--pie-chart'],
+  'with-transition': true,
+  'with-arc-labels': false,
+
+  didInsertElement() {
+    this._super(...arguments);
+    Ember.run.scheduleOnce('afterRender', this, 'renderChart');
```

Comment:
a mixin or a base component class that we extend (the latter is what I'm doing on the axes refactor branch). It seems like base component pattern is a bit more Ember-esque; what do you think?